jdk/src/solaris/bin/java_md_solinux.c

```diff
@@ -380,6 +386,14 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                                            asked for? Current model is
                                            fine unless another model
                                            is asked for */
+#ifdef AIX
+    const char *mallocOptionsName = "MALLOCOPTIONS";
+    const char *mallocOptionsValue = "multiheap,buckets";
+    if (setenv(mallocOptionsName, mallocOptionsValue, 0) != 0) {
+      fprintf(stderr, "mallocOption failed: performance may be affected\n");
+      fflush(stderr);
```

Comment:
Please fix the indentation (8 spaces here).
`fflush` is not needed for `stderr`.
I suggest the message should have more information: e.g.
> setenv('MALLOCOPTIONS=multiheap,buckets') failed: performance may be affected

Reply:
sure, updated